test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -203,6 +203,15 @@ describe('POST /group/:groupId/join', () => {
         await expect(invitedUser.get('/user')).to.eventually.have.nested.property('party._id', party._id);
       });
 
+      it('accepting a redundant party invite will let the user stay in the party', async () => {
```

Comment:
it's looking good @JalanshMunshi ! Just one last thing, can we add a reference to the fixed issue in the test title so something like `- issue #12291 ` at the end?

Reply:
@paglias Yeah sure. Like this? 

`Issue #12291: accepting a redundant party invite will let the user stay in the party'

Or add it in a comment just above the title?

Reply:
`Issue #12291: accepting a redundant party invite will let the user stay in the party ` is perfect 👍

Reply:
Done. Resolving this conversation.